{{cookiecutter.project_name}}/.gitignore

```diff
@@ -3,6 +3,13 @@
 *~
 buck-out/
 
+{% if not cookiecutter.ide_integration -%}
```

Comment:
Out of convenience, I would just always add these to the `.gitignore`. Even if the person setting up the project might not be using them, other contributors will.

Reply:
Oh wait I think I got the logic the wrong way around. Now we're keeping them, right?

Reply:
I always commit them actually.